roles/matrix-cactus-comments/defaults/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Cactus Comments is a federated comment system built on Matrix
+
+matrix_cactus_comments_enabled: true
+matrix_cactus_comments_container_image_self_build: false
+matrix_cactus_comments_docker_repo: "https://gitlab.com/cactus-comments/cactus-appservice.git"
+matrix_cactus_comments_docker_repo_version: "{{ matrix_cactus_comments_version if matrix_cactus_comments_version != 'latest' else 'main' }}"
+matrix_cactus_comments_docker_src_files_path: "{{ matrix_cactus_comments_base_path }}/docker-src"
+
+
+matrix_cactus_comments_base_path: "{{ matrix_base_data_path }}/cactus-comments"
+matrix_cactus_comments_container_tmp_path: "{{ matrix_cactus_comments_base_path }}/tmp"
+
+matrix_cactus_comments_app_service_config_file: "{{ matrix_cactus_comments_base_path }}/cactus_appservice.yaml"
+matrix_cactus_comments_app_service_env_file: "{{ matrix_cactus_comments_base_path }}/cactus.env"
+
+matrix_cactus_comments_as_token: ''
+matrix_cactus_comments_hs_token: ''
+matrix_cactus_comments_homeserver_url: ''
+matrix_cactus_comments_user_id: "cactusbot"
```

Comment:
We usually name bots `bot.something`. It will be more consisent if we go for that.

The documentation will also need to be adjusted.

Reply:
True. Thought that provoke problems later on but that was a misunderstanding of mine